.gitlab/deps_fetch/deps_fetch.yml

```diff
@@ -106,3 +106,15 @@ go_e2e_deps:
         prefix: "go_e2e_deps_modcache"
       paths:
         - modcache_e2e.tar.xz
+
+fetch_openjdk:
+  needs: []
+  tags: ["arch:amd64"]
+  rules:
+    - !reference [.manual]
+  stage: deps_fetch
+  image: registry.ddbuild.io/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  script:
+    - wget https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.25%2B9/OpenJDK11U-jre_x64_windows_hotspot_11.0.25_9.zip -O OpenJDK11U-jre_x64_windows_hotspot_11.0.25_9.zip
+    - echo '052f09448d5b8d9afb7a8e5049d40d7fafa8f5884afe6043bb2359787fd41e84  OpenJDK11U-jre_x64_windows_hotspot_11.0.25_9.zip' | sha256sum -c
+    - $S3_CP_CMD OpenJDK11U-jre_x64_windows_hotspot_11.0.25_9.zip $S3_DD_AGENT_OMNIBUS_JAVA_URI/OpenJDK11U-jre_x64_windows_hotspot_11.0.25_9.zip --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏For maintenance it would be way easier to have the version (`11.0.25_9`) and the sha (`052f09448d5b8d9afb7a8e5049d40d7fafa8f5884afe6043bb2359787fd41e84`) in variables for easier version bumps. WDYT?